src/qt/walletcontroller.cpp

```diff
@@ -340,7 +349,11 @@ LoadWalletsActivity::LoadWalletsActivity(WalletController* wallet_controller, QW
 
 void LoadWalletsActivity::load()
 {
-    showProgressDialog(tr("Loading wallets…"));
+    showProgressDialog(
+      //: Title of progress window when which is displayed when wallets are being loaded.
+      tr("Load Wallet"),
```

Comment:
as already mentioned by @hebasto [here](https://github.com/bitcoin-core/gui/pull/409/files#r720722285)
```suggestion
      tr("Load Wallets"),
```

Reply:
Addressed in ccecfa8